web/menu.jspf

```diff
@@ -202,10 +204,12 @@ org.opensolaris.opengrok.web.Util"
     </tbody>
 </table>
 <div id="form-controls">
+    <input tabindex="9" class="submit btn" type="button" value="Search All Projects" onclick="javascript: SearchAll();"/>
     <input tabindex="9" class="submit btn" type="submit" value="Search"/>
     <input tabindex="10" class="submit btn" onclick="javascript: clearSearchFrom();"
            type="button" value="Clear"/>
     <a href="help.jsp" target="_blank" class="help btn" title="Help" tabindex="11" />Help</a>
+	<input type="hidden" id="hdnSearchAll" name="hdnSearchAll" value="false">
```

Comment:
The name is actually a little clumsy here because it is a part of an URL which you can send to any user. 

I'd prefer `all` or `search_all` or `searchAll`.